src/main/resources/react/src/redux/auth/action.js

```diff
@@ -1,15 +1,15 @@
 import { types } from './types';
 
-export const chooseLanguage = (lang) => ({
+export const chooseLanguage = ( lang ) => ({
```

Comment:
It would be better for all to select one version of IDE formatter for preventing such commits into the future.